modules/mesaides/templates/mesaides_static.erb

```diff
@@ -0,0 +1,41 @@
+location /favicon.ico {
```

Comment:
It is a pity to have all thoses rules. Static files should probably be in a `static` subfolder with a single rule. Next iteration, I guess.